src/main/java/net/minecraftforge/client/model/ModelLoaderRegistry.java

```diff
@@ -193,6 +193,27 @@ else if(VanillaLoader.INSTANCE.accepts(actual))
         return model;
     }
 
+    public static boolean isCustomModel(ResourceLocation location)
+    {
+        ResourceLocation actual = getActualLocation(location);
+        for(ICustomModelLoader loader : loaders)
+        {
+            try
+            {
+                if(loader.accepts(actual))
+                {
+                    return true;
+                }
+            }
+            catch(Exception e)
+            {
+                LOGGER.warn("Exception checking if model {} can be loaded with loader {}: {}", location, loader, e);
```

Comment:
You don't need a "{}" placeholder to log an exception with log4j. If you add a placeholder for an exception, the stacktrace will not be logged.